README.md

```diff
@@ -1,2 +1,179 @@
+- **Integration of navigability with links and "onclick" events integrated with JS**
+- **Pop-up menu used in Mobile version**
+- **"More" button used for displaying full list of bands"**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [https://alexansaa.github.io/ConcertAdv/](#LiveDemo)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- You need to have NodeJS installed
+- A Web Browser (Google Chrome, Firefox, etc)
+- A Code Editor (Notepad++, VSCode, etc)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+git clone yourlink repo https://github.com/alexansaa/ConcertAdv.git
+
+cd your folder name
+```
+
+### Install
+
+You won't need further instalation
+
+### Usage
+
+To run the project, execute the following command:
+
+Double-click on the index file (index.html)
+
+### Run tests
+
+N/A
+
+### Deployment
+
+You can deploy this project using:
+
+Deploy this project on any websever
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Kleber Saavedra**
+
+- GitHub: [@alexansaa](https://github.com/alexansaa)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add Two web pages: Home & About**
+- [ ] **Add Responsive Design implemented**
+- [ ] **Add Two versions where developed: Mobile & Desktop. Breakpoints of viewport implemented with Media Queries**
+- [ ] **Add JavaScript used for dinamyc implementation of elements within an JS object**
+- [ ] **Add Integration of navigability with links and "onclick" events integrated with JS**
+- [ ] **Add Pop-up menu used in Mobile version**
+- [ ] **Add "More" button used for displaying full list of bands"**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give me a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+The design behind the web pages was developed by [Cindy Shin](https://www.behance.net/adagio07) and can be found on [Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [CC BY-NC 4.0](https://creativecommons.org/licenses/by-nc/4.0/) licensed.
```

Comment:
[OPTIONAL] Kindly add a proper GitHub license link as you have done in previous milestones and ensure it is personalized to this project as this gives your project a more professional look👍.